playersScore.js

```diff
@@ -0,0 +1,131 @@
+const player1Name = document.getElementById("p1-input");
+const player2Name = document.getElementById("p2-input");
+const startBtn = document.getElementById("start-btn");
+const playersSection = document.getElementById("players-container");
+const scoreContainer = document.getElementById("score");
+const restartBtn = document.getElementById("restart-btn");
```

Comment:
Very nice coding style to find all these elements already at the start, makes it clear which elements you will be working with later in the file 🧠